definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-share-a-case.json

```diff
@@ -0,0 +1,51 @@
+[
+  {
+    "LiveFrom": "18/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "PetitionerOrganisationPolicy",
+    "UserRole": "caseworker-divorce-solicitor",
```

Comment:
I actually wonder if this should not be [PETSOLICITOR]?

Reply:
The new CaseField is called `PetitionerOrganisationPolicy`

Reply:
On the "SolicitorCreate" event the "[PETSOLICITOR]" role is not yet set so it would have to be "caseworker-divorce-solicitor"